test/StressTests/Server/Documents/PeriodicBackup/Retention.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Threading.Tasks;
```

Comment:
Why this is commented out?

Reply:
didnt notice it, I have to add stress tests for azure as well